tests/test_create.py

```diff
@@ -20,11 +21,16 @@
         mosaic_content["tiles"][qk] = [os.path.join(basepath, item) for item in asset]
 
 
+def _filter_and_sort(*args, **kwargs):
+    dataset = default_filter(*args, **kwargs)
+    return sorted(dataset, key=lambda x: x["properties"]["path"], reverse=True)
```

Comment:
You need to use `accessor(x)` here.

Reply:
Or sort within the filter

Reply:
I mean sure, but this is just for this test.

Reply:
Oh sorry I didn't see this was a test